### DISCUSSION TITLE
improvements for firestick version

### DISCUSSION THREAD

**Discussion by foxyloxy-eng** [GerryDazoo/SlingerPlayer#12] [Ideas]
Can we get an option to bring up a remote window like in the windows version?

**Comment by as400jockey** [comment]
I love this gadget. Finally - slingbox playing correctly on my old plasma HD TV!

I will run the remote from my android phone or tablet... can't picture how a remote would work on the firestick.. took me about 20 minutes to get the url for downloading the app onto the screen.

I currently have 5 cable boxes at $10/ea/mo and only myself and my wife here. Slingbox and the slingbox firestick app (or was it the Roku app) was supposed to allow me to get rid of a box or two. Until Sling killed the app on one of those two. Then I bought the other, and they killed that one too. Then they killed the android app and finally the entire thing.

So this works beautifully for watching from my 15 year old plasma TV.
